README.md

```diff
@@ -137,6 +137,10 @@ If for some reason you don't want to include some commits in your rebase PR, you
 ...
 ```
 
+## Manual Override
+
+Sometimes it is necessary to have the bot skip making a pull request in a repository without modifying a continuous integration configuration file. In these cases you may add the label `rebase/manual` to **any** pull request in a repository and the bot will skip making a rebase pull request. If Slack integration is enabled the bot will broadcast a message that it has found the `rebase/manual` label when it runs.
```

Comment:
To any pull request? Does that it doesn't have to be the PR that is for the rebase? That surprises me, I don't think that was the intention of the feature

CC @odvarkadaniel

Reply:
We look for this label in a destination branch: https://github.com/openshift-eng/rebasebot/blob/main/rebasebot/bot.py#L466. If we find a PR within the destination branch with the label `rebase/manual` we terminate the Rebasebot `return True`.
I think that the wording that Mike provided in this PR is correct.

Reply:
Right, is that what we want? So I could apply the label to a random PR and that would suddenly stop my rebase from syncing? I wonder if it's better to have the label apply only to an existing PR created by the bot? ie from the scratch repo, source branch, into the destination repo, main branch?

Reply:
i wasn't sure about the intended behavior, but this was my understanding based on the code in https://github.com/openshift-eng/rebasebot/blob/main/rebasebot/bot.py#L430-L436

Reply:
I had thought we wanted it to only be on the rebase PR itself that it would check, I hadn't realised the change while reviewing, so perhaps we can get an update to the logic to match the original intention (if others agree that's the right thing to do)

Reply:
i think we should probably talk about it, i'm wondering if this style was chosen to avoid the complexity around the bot attempting to find previous PRs it has opened or something?

Reply:
It was my understanding that the bot already has logic to go and find a PR it created no? Else how would it know not to create a new PR when one already exists?

Reply:
i'm not 100% sure, i saw a repo where the bot had a couple PRs open so that's why i thought maybe there was a difference.